tfjs-backend-webgpu/src/kernels/select_webgpu.ts

```diff
@@ -17,33 +17,34 @@
 import {util} from '@tensorflow/tfjs-core';
 
 import {getCoordsDataType} from '../shader_preprocessor';
+import {getWorkGroupSizeStringWgsl} from '../shader_preprocessor_wgsl';
 import {computeDispatch, flatDispatchLayout} from '../webgpu_util';
 
-import {WebGPUProgram} from './webgpu_program';
+import {getUseWgsl, WebGPUProgram} from './webgpu_program';
 
 export class SelectProgram implements WebGPUProgram {
   variableNames = ['c', 'a', 'b'];
   outputShape: number[];
   shaderKey: string;
   dispatchLayout: {x: number[]};
   dispatch: [number, number, number];
-  workPerThread = 4;
   workGroupSize: [number, number, number] = [16, 1, 1];
```

Comment:
Please change the work group size to [64, 1, 1] instead of [16, 1, 1]

Reply:
done